pandas/io/json/_json.py

```diff
@@ -757,13 +740,8 @@ def close(self):
 
         If an open stream or file was passed, we leave it open.
         """
-        if self.should_close:
-            try:
-                self.open_stream.close()
-            except (OSError, AttributeError):
-                pass
-        for file_handle in self.file_handles:
-            file_handle.close()
+        if self.handle_args is not None:
+            self.handle_args.close()
```

Comment:
that was previously in a try-except block? Should we add try-except for `IOArgs.close`/`IOHanldeArgs.close`?

Reply:
yeah i think you need the try/except in the .close itself